### DISCUSSION TITLE
Super user

### DISCUSSION THREAD

**Discussion by kaiykay** [community/community#40966] [Repositories]
### Select Topic Area

Bug

### Body

How can this repo be treated as normal root or as it supports super user

**Comment by ossanna16** [comment]
Hi there @kaiykay and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by DSDmark** [comment]
Hi @kaiykay,
   
> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

Or why any chance if you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc).  its a video. so, just ***sit back and relax***

and why the way,

it's an official **GitHub Community** repository, as you can see there [**GitHub**](https://github.com/community). It's a normal repository like yours repository. But, it's creation for community to update, report, improvement interface, etc. if You have something that can improve GitHub or some idea which you think is great then you can request here to implement that idea by creating discussion, or you can report issue. If you're facing in GitHub or it's products.
